SFPortfolioApp/SFPortfolioApp/Components/BottomSheet/BottomSheetView.swift

```diff
@@ -67,22 +67,19 @@ extension BottomSheetView {
                 content()
                     .background(measurerView)
             }
-            .frame(width: UIScreen.main.bounds.width, height: height)
+            .frame(width: UIScreen.main.bounds.width, height: height + safeAreaHeight)
```

Comment:
Wouldn't it be more appropriate if we rename safeAreaHeight to bottomSafeAreaHeight?

Reply:
Renamed